src/main/kotlin/com/velocitypowered/api/language/kotlin/ext/GameProfileExt.kt

```diff
@@ -0,0 +1,27 @@
+@file:Suppress("NOTHING_TO_INLINE")
```

Comment:
If there's nothing to inline, don't inline it...

Reply:
From documentation:

```
If an inline function has no inlinable function parameters and no reified type parameters, the compiler will issue a warning, since inlining such functions is very unlikely to be beneficial (you can suppress the warning if you are sure the inlining is needed using the annotation @Suppress("NOTHING_TO_INLINE") ).
```

In this case GameProfile.plus actualy inlines, Its can be check by using kotlin bytecode viewer:

![image](https://user-images.githubusercontent.com/18630111/118404522-f3a76a80-b67b-11eb-98b5-6091ce6d3559.png)

![image](https://user-images.githubusercontent.com/18630111/118404528-fc983c00-b67b-11eb-8ce6-fbbc1a6dfe75.png)

Reply:
I know it still inlines, but there's no reason for it. You might as well just call it properly and let the JVM handle optimisations such as that at runtime, which it indeed does when it's a simple delegate method.